### DISCUSSION TITLE
Proxy access to internet

### DISCUSSION THREAD

**Discussion by AARON8888** [FlareSolverr/FlareSolverr#957] [Q&A]
I have a question.

I use proxy connection to access to internet settled in my windows system, and VPN in my adapter connection previously.

When I run FlareSolverr to bypass cloudflare protection in a site, is FlareSolverr app bypassing my proxy connection in my SO and connected to my VPN?

I know some apps for Windows like Firefox or FileZilla FTP Client can bypass proxy connection which is deactivated by default and then use my connection adapter. But generally apps use the LAN connection of my SO.

Thanks in advance.

**Comment by ilike2burnthing** [comment]
Whatever your adapter or system is configured to use, FlareSolverr will use.
